docs/Changelog.md

```diff
@@ -14034,11 +14040,17 @@ This version of the operator has been available since version 12 of the default
 
 ### <a name="Dropout-12"></a>**Dropout-12**</a>
 
-  Dropout takes one input floating tensor and produces two tensor outputs,
-  output (floating tensor) and mask (`Tensor<bool>`). Depending on whether it is
-  in test mode or not, the output Y will either be a random dropout, or a simple
-  copy of the input. Note that our implementation of Dropout does scaling in
-  the training phase, so during testing nothing needs to be done.
+  Dropout takes an input floating tensor and an input ratio (float scalar), and produces two tensor outputs,
+  output (floating tensor) and mask (`Tensor<bool>`). The output Y will be a random dropout;
+  Note that our implementation of Dropout does scaling in
+  the training phase, so during testing nothing needs to be done. The output is computed as :
+  ```
+  output_i = scale * input_i * mask_i,
```

Comment:
What is `_i`? Could you please give it a formal definition? The input name below is `data`. Could you please change `input_i` to `data_i` for consistency? Thanks.